### ISSUE TITLE
How to increase cache size?

### ISSUE THREAD

**Issue by phdru** [mfenniak/pg8000#128]
https://github.com/mfenniak/pg8000/blob/c37b9b0a52ef1846f86adfc363f2298cac4dff2e/pg8000/core.py#L2024:
```python
                raise InterfaceError(
                    "With autocommit on, it's not possible to retrieve more "
                    "rows than the pg8000 cache size, as the portal is closed "
                    "when the transaction is closed.")
```
Can I programmatically increase the cache size? If yes — how? Set Connection._row_cache_size?

**Comment by phdru** [comment]
Ok, got it — I implemented setting cache size in #130 and it really helps.

**Comment by tlocke** [comment]
We now read all rows from the db in one go, so everything's cached, so that should sidestep this problem. See https://github.com/mfenniak/pg8000/commit/eb41aab4a93446b863a450dfd20c91ace54b5488.
